frontend/simple/index.html

```diff
@@ -21,6 +21,7 @@ <h1 class="title">Outdated browser</h1>
     <div id="app">
       <nav-bar></nav-bar>
       <router-view></router-view>
+      <modal></modal>
```

Comment:
GitHub highlights this in red and I don't know why. Maybe it's a reserved HTML tag?

Reply:
I've found it strange too, my editor is fine, Vue doesn't give any error... not sure why 🤔